src/js/bootstrap.js

```diff
@@ -57,8 +59,7 @@ const App = () => {
       const { chrome: chromeConfig } = data;
       dispatch(loadModulesSchema(data));
       initializeAccessRequestCookies();
-      const libjwt = libjwtSetup(chromeConfig);
-      libjwt.initPromise.then(() => setJwtState(true));
+      const libjwt = libjwtSetup(chromeConfig, setJwtState);
```

Comment:
I am not a big fan of sending callback state set function to a rather complex multi promise function. How about we'd change the `libjwtSetup` function to be
```JS
const libjwtSetup = (libjwt) => {
  return libjwt.initPromise.then(() => {
    return libjwt.jwt
      .getUserInfo()
      .then((...data) => {
        console.log('sentry step');
        sentry(...data);
      })
      .catch(noop);
  });
};
```

And were to change the `useEffect` to
```JS
const libjwt = auth(chromeConfig || {});
libjwtSetup(libjwt).then(() => setReadyState(true));
```